features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/RumFeature.kt

```diff
@@ -379,6 +380,24 @@ internal class RumFeature(
         }
     }
 
+    /**
+     * Enables the tracking of JankStats for the given activity. This should only be necessary for the
+     * initial activity of an application if Datadog is initialized after that activity is created.
+     * @param activity the activity to track
+     */
+    internal fun enableJankStatsTracking(activity: Activity) {
+        try {
+            jankStatsActivityLifecycleListener?.onActivityStarted(activity)
+        } catch (@Suppress("TooGenericExceptionCaught") e: Exception) {
+            sdkCore.internalLogger.log(
+                InternalLogger.Level.ERROR,
+                InternalLogger.Target.TELEMETRY,
+                { FAILED_TO_ENABLE_JANK_STATS_TRACKING_MANUALLY },
+                e
+            )
+        }
```

Comment:
which exception do we expect here? or it is just a generic safeguard, just in case?

Reply:
Generic safeguard. Detekt complained about it, and instead of suppressing it I thought it would be better to catch and log.

Reply:
To be clearer, Detekt complained about `onActivityStarted` being unsafe.